subprojects/base-services/src/main/java/org/gradle/internal/hash/Hashing.java

```diff
@@ -196,4 +278,91 @@ public void putHash(HashCode hashCode) {
             putBytes(hashCode.getBytes());
         }
     }
+
+    @VisibleForTesting
```

Comment:
`@VisibleForTesting` is not necessary any more.

Reply:
Good catch, made it private.